### PR TITLE
chore(tag): remove _createdAt and _updatedAt from Tag entity

### PR DESCRIPTION
**Pull request by sri-kirsh**
## Goal
- remove _createdAt and _updatedAt from Tag entity

## Why?
- these fields are not being used by any clients
- however tag's time_created and time_updated in the database contains null values, so some of our queries had to do `min/max` of tags and causes database performance degradation, especially for users with larger tags (e.g 600k, 1M+ tags)

Ticket: https://getpocket.atlassian.net/browse/INFRA-1058

slack message: https://pocket.slack.com/archives/C03QVL4SU/p1677005467804549

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ services:
       - TZ=US/Central
 
   localstack:
-    image: localstack/localstack:1.2
+    image: localstack/localstack:1.3
```
Comment (sri-kirsh) [inline docker-compose.yml:17]: breaks local test - so updated to next version

more info here: https://pocket.slack.com/archives/C03QVL4SU/p1677016106673369

File: src/dataService/tagDataService.ts [hunk 2]
```diff
@@ -130,8 +125,8 @@ export class TagDataService {
       .pluck('tag');
 
     const tags = await this.getTagsByUserSubQuery()
-      .whereIn('tag', latestTags)
-      .orderBy('_updatedAt', 'desc');
+      .whereIn('tag', latestTags);
+      //.orderBy('_updatedAt', 'desc');
```
Comment (sri-kirsh) [inline src/dataService/tagDataService.ts:129]:
```suggestion
```

File: src/test/graphql/queries/suggested-tags.integration.ts [hunk 3]
```diff
@@ -127,8 +126,8 @@ describe('tags query tests - happy path', () => {
       const tags = res.body.data?._entities[0].savedItemById.suggestedTags;
       expect(tags.length).to.equal(3);
       expect(tags[0].name).to.equal('adventure');
-      expect(tags[1].name).to.equal('thriller');
-      expect(tags[2].name).to.equal('romance');
+      expect(tags[1].name).to.equal('romance');
```
Comment (sri-kirsh) [inline src/test/graphql/queries/suggested-tags.integration.ts:129]: coz we removed `order by updatedAt` for suggestedTags, the order is different but the top 3 values are same

**Review by cmharlow — APPROVED** [review]
Thanks for doing this!

**Pull request merged**